e2e-tests/gef/cypress/fixtures/mocks/mock-deals.js

```diff
@@ -213,6 +213,7 @@ export const MOCK_APPLICATION_AIN = {
 };
 
 export const MOCK_APPLICATION_MIN = {
+  version: 0,
```

Comment:
Add some comments why this is set to `0` as all new deals are automatically set to `1` for FED.

Reply:
I have removed as it is not needed anymore